src/styles/globals.css

```diff
@@ -13,28 +13,28 @@
     /* Typography */
     html,
     body {
-        @apply font-systemSans text-base font-normal tracking-normal antialiased;
+        @apply font-sans text-base font-normal tracking-normal antialiased;
     }
     h1 {
-        @apply font-sans text-6xl font-semibold tracking-normal md:text-7xl md:tracking-tightest;
+        @apply font-sans text-6xl font-semibold tracking-tighter md:text-7xl md:tracking-tightest;
```

Comment:
What is "md:text-7xl"? Is that for mobile? If so, I think we need to tweak. Lets have mobile also be 6xl but keep the different tracking.